File: Hero/ClientApp/src/app/models/sanctions/SanctionsCheckRequest.ts
import { SanctionStage } from "@app/enums/SanctionStage";

export interface SanctionsCheckRequest {
    clientName: string;
    clientUid: string;
    clientId: number;
    countryIsoCode: string;
    stage: SanctionStage;
    isSendEmail: boolean;
    onGoingScreening?: boolean;
}

File: Hero/Models/SanctionsCheckRequest.cs
using System.Collections.Generic;

namespace Hero.Models;

public record SanctionsCheckRequest
{
    public string ClientUid { get; init; }
    public string ClientName { get; init; }
    public int ClientId { get; init; }
    public string CountryIsoCode { get; init; }
    public string Stage { get; init; }
    public bool IsSendEmail { get; init; }
    public bool? onGoingScreening {get; init;}
}
